### PR TITLE
Lesson9

### PR DESCRIPTION
**Pull request by vtx123**

### CODE REVIEW

File: Lesson9/Lesson9_1.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+void translateArray(int numbers[], int size){
+    for(int i = 0; i < size; i++){
+        if(numbers[i] > 0){
+            std::cout << numbers[i] * 2 << " ";
+        }else {
+            std::cout << "0 ";
+        }
+    }
+}
```
Comment (bohdanpc) [inline Lesson9/Lesson9_1.cpp]: not translated output but modification of original array

File: Lesson9/Lesson9_3.cpp [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// Lesson 9 Issue 3
+#include <iostream>
+
+bool isPalindrom(const char str[]){
+    unsigned lenght = static_cast<int>(strlen(str));
```
Comment (bohdanpc) [inline Lesson9/Lesson9_3.cpp]: length 😅

Comment (vtx123) [reply Lesson9/Lesson9_3.cpp]: Uhh... always struggling with it 😅

Comment (vtx123) [reply Lesson9/Lesson9_3.cpp]: fixed

File: Lesson9/Lesson9_1.cpp (hunk 1 above)
Comment (vtx123) [reply Lesson9/Lesson9_1.cpp]: fixed, find some additional bugs)
and have a question for next lesson

**Pull request merged**